packages/api/src/external/fhir/document/index.ts

```diff
@@ -79,12 +81,8 @@ export const toFHIR = (
       ]
     : [];
 
-  const containedContent: Resource[] = [
-    {
-      resourceType: "Patient",
-      id: patient.id,
-    },
-  ];
+  // https://www.hl7.org/fhir/R4/domainresource-definitions.html#DomainResource.contained
+  const containedContent: Resource[] = [];
```

Comment:
Looking at `contained` we see that we should only include there resources that are not available through the FHIR server.